runtime/autoload/health.vim

```diff
@@ -9,13 +9,20 @@ function! s:enhance_syntax() abort
   highlight link healthInfo ModeMsg
 
   syntax keyword healthSuccess SUCCESS
-  highlight link healthSuccess Function
+  highlight link healthSuccess ModeMsg
 
   syntax keyword healthSuggestion SUGGESTIONS
   highlight link healthSuggestion String
 
+  syntax match healthHelp "|.\{-}|" contains=healthBar
+  syntax match healthBar  "|" contained conceal
+  highlight link healthHelp Identifier
+
   " We do not care about markdown syntax errors in :CheckHealth output.
   highlight! link markdownError Normal
+
+  " We don't need code blocks.
+  syntax clear markdownCodeBlock
```

Comment:
So.. I can keep going with the `syntax clear markdownCodeBlock`? Or redefine `markdownCodeBlock` in `s:enhance_syntax()`?